internal/view/issue.go

```diff
@@ -457,3 +458,149 @@ func (i Issue) renderPlain(w io.Writer) error {
 	_, err = fmt.Fprint(w, out)
 	return err
 }
+
+// printableIssue is a jira.Issue that can be marshaled into a format of choice f.e. JSON and
+// printed for other programs to consume.
+type printableIssue struct {
+	Key    string               `json:"key"`
+	Fields printableIssueFields `json:"fields"`
+}
+
+type printableIssueFields struct {
```

Comment:
Instead of sending few fields, why not output the exact response from the API? I am pretty sure there are usecases that require fields that we haven't considered in the response.